hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/reload/jvm/tooling/sidecar/DtReloadCounterStatusItem.kt

```diff
@@ -38,12 +40,18 @@ fun DtCollapsedReloadCounterStatusItem() {
     val state = ReloadCountState.composeValue()
     if (state.successfulReloads < 1) return
 
-    val defaultText = "⟳${state.successfulReloads}"
-    val defaultModifier = Modifier.tag(ReloadCounterText)
-    val (text, modifier) = when {
-        state.successfulReloads < 10 -> defaultText to defaultModifier
-        state.successfulReloads < 100 -> defaultText to defaultModifier.scale(0.9f)
-        else -> defaultText.drop(1) to defaultModifier.scale(0.7f)
+    val textScale = when {
+        state.successfulReloads < 10 -> 1.0f
+        state.successfulReloads < 100 -> 0.9f
+        else -> 0.7f
+    }
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier.scale(textScale)
+    ) {
+        if (state.successfulReloads < 100) {
+            DtText("⟳", modifier = Modifier.scale(1.25f))
```

Comment:
😬 I think you should use the same icon as in `DtReloadCounterStatusItem`

Reply:
Switched to the default icon. I had to scale down the text a little bit to make it fit (this is why I originally opted to use the unicode character). But I think it is still readable.

Reply:
I would suggest using the size modifier in this case, but I am ok for landing this